core/src/main/java/org/fao/geonet/kernel/security/openidconnect/GeonetworkOidcLogoutHandler.java

```diff
@@ -70,7 +72,7 @@ private URI createPostLogoutRedirectUri(HttpServletRequest request) {
             String path = servletContext.getContextPath();
             return new URI(protocol + "://" + host + ":" + port + path);
         } catch (URISyntaxException e) {
-            e.printStackTrace();
+            Log.error(Geonet.SECURITY,"ignored error parsing URI",e);
```

Comment:
Please change error as this would be very confusing to see these errors in the logs as type Error.  Maybe changed it to a Log.debug with a message indicating that parsing the url failed and it is OK to ignore.  I'm also not sure why we should ignore an error related to parsing the url - so it may be good to add a comment.

Reply:
I changed it to a debug message, but I'm not sure if it can just be ignored.  This would indicate that the logout message was not sent to the remote OIDC service - which is, i think (not sure), is against the spec.
This error shouldn't happen - it indicates that the OIDC configuration is problematic.
If this error occurs, the sys admin should definitely investigate.